packages/hdwallet-gridplus/src/gridplus.ts

```
@@ -8,8 +8,8 @@ import * as eth from "./ethereum";
 import * as mayachain from "./mayachain";
 import * as solana from "./solana";
 import * as thorchain from "./thorchain";
-import { GridPlusTransport } from "./transport";
-import { convertXpubVersion, scriptTypeToAccountType } from "./utils";
+
+const ZERO_BUFFER = Buffer.alloc(32)
```

ESLint

[error] Insert `;`

(prettier/prettier)